app/components/Wallet/useWallet.tsx

```diff
@@ -10,6 +11,26 @@ import Wallet, {
 	getBalanceMethodName,
 	getNetworkMethodName,
 } from "sats-connect";
+import axios from "axios";
+
+interface UtxoI {
+	scriptpubkey: string;
```

Comment:
```suggestion
	script_pubkey: string;
```

Reply:
The API response is received with the key `scriptpubkey ` so we have keep it as it is.

Reply:
ah, OK. So this should be a type, not interface. And add appropriate comment to the type.

Reply:
Done